browser-test/src/support/admin_questions.ts

```diff
@@ -179,9 +179,6 @@ export class AdminQuestions {
 
     expect(tableInnerText).toContain(questionName)
     expect(tableInnerText).toContain(questionText)
-    expect(
```

Comment:
Should this assertion be changed to something else?

Reply:
I don't think so. The fact that it's active is being tested by the presence of "New version" button. I think it's enough. And it will change soon anyway as we going to render both active and draft rows.